### PR TITLE
Improve explanations of Tangle, addresses, private keys, bundles, and…

### PR DESCRIPTION
**Pull request by JakeSCahill**
… transactions

### CODE REVIEW

File: getting-started/0.1/introduction/what-is-a-transaction.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
 # What is a transaction?
 
-**A transaction is an instruction that sends data or IOTA tokens to an address or debits IOTA tokens from an address.**
+**A transaction is a packet that sends messages, withdraws IOTA tokens from an address, or deposits IOTA tokens to an address.**
```
Comment (l3wi) [inline getting-started/0.1/introduction/what-is-a-transaction.md]: This seems to describe a Bundle not a TX. TXs are single operations that can stand alone (Data) or be bundled to have more complex actions (Tokens from A to B)

File: getting-started/0.1/introduction/what-is-a-bundle.md [hunk 2]
```diff
@@ -22,9 +22,9 @@ Your balance is distributed among three addresses:
 
 To send 100Mi to recipient A, you must create the following transactions and send them to an IRI node as a bundle:
 
-* **Input transaction:** Debit 100Mi from my balance and check the signature to verify that I own those IOTA tokens
-* **Output transaction:** Credit 100Mi to the recipient's address
-* **Output transaction:** To avoid spending from address 2 again, transfer the remaining 5Mi from address 2 to address 3
+* **Input transaction:** Withdraw 100Mi from my address and check the signature to verify that I own it
```
Comment (l3wi) [inline getting-started/0.1/introduction/what-is-a-bundle.md]: Like the clarification in language 👍

File: iota-basics/0.1/concepts/addresses-and-signatures.md [hunk 3]
```diff
@@ -1,19 +1,23 @@
 # Addresses and signatures
 
-**Seeds are used to create private keys, and in turn, private keys are used to create addresses and signatures. Addresses are public and can safely be shared among others in the network. Signatures are cryptographic proof of ownership of an address. So, when a transaction contains a valid signature, it was created by the owner of the correct seed.**
+**Each client in an IOTA network has a secret password called a seed, which is used to create addresses and signatures. Addresses are the accounts from which transactions are sent and received, and signatures prove ownership of an address.**
```
Comment (l3wi) [inline iota-basics/0.1/concepts/addresses-and-signatures.md]: Great section 👌

File: iota-basics/0.1/concepts/bundles-and-transactions.md [hunk 4]
```diff
@@ -1,24 +1,32 @@
...
+You're at an online checkout and the total to pay is 10Mi. Your seed has 2 addresses (index 0 and 1), which both contain 5Mi. So, you create three transactions: One input transaction to debit 5Mi from address 0, another input transaction to debit 5Mi from address 1, and one output transaction to credit 10Mi to the vendor's address. (We'll assume that both addresses in the input transactions were created from a private key with security level 1, so the signatures can fit in each transaction.)
+
+For the vendor to receive 10Mi, all three of those transactions must be valid. They're sequential instructions that rely on each other's validity to achieve the overall goal of transferring IOTA tokens.
+
+**Note:** It's not just multiple transactions that need to be packaged in a bundle, even individual ones do.
 
 ## Input and output transactions
 
-A bundle can consist of any number of input and output transactions. We recommend a maximum of 30 transactions in a bundle.
+A bundle can consist of any number of input and output transactions, however we recommend a maximum of 30 in a bundle.
```
Comment (l3wi) [inline iota-basics/0.1/concepts/bundles-and-transactions.md]: Why? 

I'd add `in a bundle due to the Proof of Work overhead`. (Add link to PoW)

File: iota-basics/0.1/introduction/overview.md [hunk 5]
```diff
@@ -2,28 +2,43 @@
...
 
-The IOTA [client libraries](root://client-libraries/0.1/introduction/overview.md) provide you with all the tools you need, including those to create addresses, signatures, and transactions. All the methods that handle seeds are executed on the client side (your computer), to make sure that they're never sent anywhere insecure.
+**Tip:** The IOTA [client libraries](root://client-libraries/0.1/introduction/overview.md) provide you with all the tools you need, including those to create addresses, signatures, and bundles. All the methods that handle seeds are executed on the client side (your computer), to make sure that they're never sent anywhere insecure.
 
-## IRI nodes
+## Trust
 
-Clients must send bundles to IRI nodes, which are responsible for [validating all transactions](root://iri/0.1/concepts/transaction-validation.md) and keeping an immutible history of them in ledgers. Each IRI node has its own ledger.
+You might be wondering how you can trust a node. After all, it's responsible for protecting the balance of your addresses.
```
Comment (l3wi) [inline iota-basics/0.1/introduction/overview.md]: > it's responsible for protecting the balance of your addresses
🤔 I don't understand this sentence. Users are responsible for protecting addresses.

More appropriate would be something related to you access to the network. A node can refuse you access to read or write the tangle but also modify responses to calls like getBalances which can be alarming for users.

File: iota-basics/0.1/introduction/overview.md [hunk 6]
```diff
@@ -2,28 +2,43 @@
...
+Well, IOTA is a [distributed ledger technology](root://getting-started/0.1/introduction/what-is-dlt.md). The word _distributed_ is the key. When a node receives a transaction, validates it, and appends it to its ledger, it doesn't stop there. The IOTA protocol states that all nodes must forward transactions onto other nodes, called their neighbors. This way, all nodes receive and validate all transactions, and all nodes hold a consistent, distributed ledger, removing the need to trust any individual.
 
-To be valid, transactions must conform to a [structure](../references/structure-of-a-transaction.md) in the IOTA protocol. Two of the fields in this structure are [`branchTransaction` and `trunkTransaction`](../references/structure-of-a-transaction.md), which must contain a reference to two previous transactions in an IRI node's ledger. IRI nodes select previous transactions during a process called [tip selection](root://the-tangle/0.1/concepts/tip-selection.md). When a transaction references another, it forms part of the data structure called [the Tangle](root://the-tangle/0.1/introduction/overview.md).
+## immutability
```
Comment (l3wi) [inline iota-basics/0.1/introduction/overview.md]: Caps?

File: iota-basics/0.1/references/security-levels.md [hunk 7]
```diff
@@ -1,12 +1,12 @@
-# Security levels of addresses
+# Security levels
 
-**This table displays the possible security levels of an address. The higher the security level, the larger the signature, and the more transactions that are needed to contain it.**
+**This table displays the possible security levels of a private key and address pair. The higher the security level, the larger and more secure the signature of a reused address is against brute force attacks.**
 
-The security level is the same number of key fragments and key digests that a private key has.
+A larger security level means that more computations must be done to sign a transaction and that more transactions are needed in a bundle to contain the signature. Therefore, low-powered devices may want to use security level 1, whereas a large-scale company may want to use security level 3.
```
Comment (l3wi) [inline iota-basics/0.1/references/security-levels.md]: > Therefore, low-powered devices may want to use security level 1,

Please check with your source on this. I think we should recommend lvl2 as the lowest.

File: the-tangle/0.1/concepts/proof-of-work.md [hunk 8]
```diff
@@ -47,7 +47,7 @@ Because the hash is created using the contents of the transaction, if any of the
 
 ### Minimum weight magnitude
 
-The minimum weight magnitude (MWM) is a variable that defines the number of 0s that a transaction hash must end in.
+The minimum weight magnitude (MWM) is a variable that defines the number of 9s that a transaction hash must end in.
```
Comment (l3wi) [inline the-tangle/0.1/concepts/proof-of-work.md]: Ah my bad. 0's is in trits not trytes 😂

File: getting-started/0.1/introduction/what-is-a-transaction.md (hunk 1 above)
Comment (JakeSCahill) [reply getting-started/0.1/introduction/what-is-a-transaction.md]: Thanks @l3wi, I took inspiration from your example.

File: iota-basics/0.1/concepts/bundles-and-transactions.md (hunk 4 above)
Comment (JakeSCahill) [reply iota-basics/0.1/concepts/bundles-and-transactions.md]: Great point. I've added an explanation along the same lines as your example.

File: iota-basics/0.1/introduction/overview.md (hunk 5 above)
Comment (JakeSCahill) [reply iota-basics/0.1/introduction/overview.md]: 😅 Thanks, that sentence does sound a little confused. I've used your example.

File: iota-basics/0.1/references/security-levels.md (hunk 7 above)
Comment (JakeSCahill) [reply iota-basics/0.1/references/security-levels.md]: Have changed this to level 2 for now. Will check it though.

**Pull request merged**